.github/workflows/k8s-best-practices.yml

```diff
@@ -0,0 +1,59 @@
+name: Check versioning
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - '.gitignore'
+      - 'LICENSE'
+      - '**.md'
+      - Notice.txt
+      - '.github/workflows/check-versioning.yml'
+      - '.github/workflows/check-rust.yml'
+      - '.github/workflows/run-tarpaulin.yml'
+      - '.github/workflows/run-test-cases.yml'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/CODEOWNERS'
+      - '.vscode/**'
+      - docs/**
+      - scripts/**
+      - tests/**
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '.gitignore'
+      - 'LICENSE'
+      - '**.md'
+      - Notice.txt
+      - '.github/workflows/check-versioning.yml'
+      - '.github/workflows/check-rust.yml'
+      - '.github/workflows/run-tarpaulin.yml'
+      - '.github/workflows/run-test-cases.yml'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/CODEOWNERS'
+      - '.vscode/**'
+      - docs/**
+      - scripts/**
+      - tests/**
+  release:
+    types:
+      - published
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  insights:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/reactiveops/ci-images:v11-stretch
+      env:
+        FAIRWINDS_TOKEN: ${{ secrets.FAIRWINDS_TOKEN }}
+    timeout-minutes: 5
+    steps:
+    - name: Checkout the head commit of the branch
+      uses: actions/checkout@v2
+    - name: scan repo with insights
+      run: |
+        curl -L https://insights.fairwinds.com/v0/insights-ci.sh | bash
```

Comment:
Try adding `git merge-base HEAD main` above this line - curious if that fails or not.